web/modules/custom/quiltme/src/Plugin/GraphQL/InputTypes/UserSubmissionInput.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Drupal\quiltme\Plugin\GraphQL\InputTypes;
+
+use Drupal\graphql\Plugin\GraphQL\InputTypes\InputTypePluginBase;
+
+/**
+ * Custom Input type for User Submission.
+ *
+ * @GraphQLInputType(
+ *   id = "user_submission_input",
+ *   name = "UserSubmissionInput",
+ *   fields = {
+ *     "title" = "String",
+ *     "email" = "String",
+ *     "image" = "String",
+ *     "backgroundImage" = "String"
```

Comment:
@mikeethedude Is this still accurate? I think you mentioned something that this needed to be a different type?

Reply:
Good catch, I haven't updated that to be Upload yet.

Reply:
Let's keep this in place as an option for now. We can debug and fix if there are issues if decide to use the front-end.

Reply:
Sounds good. For now this is updated and ready ish to go for that.